src/main/java/mate/academy/model/User.java

```diff
@@ -1,8 +1,8 @@
 package mate.academy.model;
 
-public class User {
-    private String email;
-    private String password;
+public class User{
```

Comment:
Java Cl checks failed. Before making a Pull Request (or a new commit to an already existing PR), you need to write the mvn clean package command in the terminal and make sure that all checks pass. After that, the mentor will be able to review the homework